CMakeLists.txt

```diff
@@ -455,6 +456,15 @@ if (NOT MSVC AND NOT APPLE AND NOT ANDROID)
     endif()
 endif()
 
+if (QUIC_LINUX_IOURING_ENABLED)
+    find_library(LIBURING NAMES liburing-ffi.so)
+    if (LIBURING)
+        message(STATUS "Found liburing: ${LIBURING}")
+    else()
+        message(STATUS "liburing not found. If build fails, install liburing")
```

Comment:
should this be a warning instead of just a status?

Reply:
Looks like I copied libnuma directly above, not XDP

Reply:
replace "STATUS" with "WARNING" and I think that'll do it